pkg/controllers/kube-apiserver.go

```diff
@@ -169,6 +168,15 @@ func (s *KubeAPIServer) configure(ctx context.Context, cfg *config.Config) error
 		return fmt.Errorf("failed to discover etcd servers: %w", err)
 	}
 
+	featureGateArgs, err := cfg.ApiServer.FeatureGates.ConvertToCLIFlags()
+	if err != nil {
+		return fmt.Errorf("failed to convert feature gates to CLI flags: %w", err)
+	}
+
+	// Build the final feature gates list
+	hardcodedFeatureGates := []string{"UserNamespacesSupport=true", "UserNamespacesPodSecurityStandards=true"}
+	featureGateArgs = append(featureGateArgs, hardcodedFeatureGates...)
```

Comment:
Could we have a conflict here? Configuration allows disabling `UserNamespacesSupport` and `UserNamespacesPodSecurityStandards`.

Reply:
This is now protected against by preventing the user to specify these values in the ushift config.

https://github.com/copejon/microshift/blob/e108194ba7642c410cfe7c95281d173682e55f94/pkg/config/apiserver.go#L206-L218